### PR TITLE
Release Network Operator v1.3.0

### PR DESCRIPTION
**Pull request by e0ne**
depends-on: kubernetes#80

### CODE REVIEW

File: example/rdma-test-pod1.yml [hunk 1]
```diff
@@ -27,8 +27,8 @@ spec:
         rdma/rdma_shared_device_a: 1
       requests:
         rdma/rdma_shared_device_a: 1
-    command:
-    - sh
+    command:    - sh
+
```
Comment (adrianchiris) [inline example/rdma-test-pod1.yml:31]: nit: no need for this change

**Review by adrianchiris — APPROVED** [review]
should we also update : https://github.com/Mellanox/network-operator/blob/master/config/samples/mellanox.com_v1alpha1_nicclusterpolicy.yaml ?

**Comment by e0ne** [comment]
> should we also update : https://github.com/Mellanox/network-operator/blob/master/config/samples/mellanox.com_v1alpha1_nicclusterpolicy.yaml ?

It will be updated in #401

**Pull request merged**